server/src/logic/middleware/recipe.middleware.ts

```diff
@@ -1,27 +1,52 @@
 import { NextFunction, Request, Response } from "express";
 import { IDatabase } from "pg-promise";
-import { ModelUtils } from "./../../logic/utils/index";
 import { ApiError, DbExtensions, ERRORS, Recipe } from "./../../models/index";
 
 export class RecipeMiddleware {
 
   private _db: IDatabase<DbExtensions>&DbExtensions;
-  private _modelUtils: ModelUtils;
 
-  constructor (db: any, modelUtils: ModelUtils) {
+  constructor (db: any) {
     this._db = db;
-    this._modelUtils = modelUtils;
   }
 
-  public updateRecipe = (req: Request): Promise<Recipe> => {
-    const newRecipe: any = req.body;
+  public newUpdateRecipes = (req: Request, res: Response): Promise<Recipe> => {
```

Comment:
With the model utils approach there was always a empty fields property in the response. I tried to remove it from the response but this had broken the update functionality. Maybe I find a way to keep the model utils way.